app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -4,88 +4,112 @@ input:-webkit-autofill{
   background-image: none !important;
 }
 
-
 form.block-form {
-  display : inline-block;
+  margin: 20px auto;
 
   label {
     color : $text-dark-grey;
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
```

Comment:
So this makes screen readers avoid them? What's the purpose of this? Wouldn't people with disabilities need those labels? Can you provide us with a link to a relevant article? I'm not well-versed is such issues, and I can't find any recommendation on this topic.